### PR TITLE
Added star/unstar option in messageActionSheet.

### PR DESCRIPTION
**Pull request by vishwesh3**
fix:#706

### CODE REVIEW

File: src/api/starMessage.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/* @flow */
```
Comment (borisyankov) [inline src/api/starMessage.js]: Not a big fan of the naming conflict with the event handlers.
Why not call the API call 'toggleMessageStarred' and set the third parameter being boolean - true = star, false = unstar?

**Comment by borisyankov** [comment]
Looks good, I had just one comment.

File: src/api/starMessage.js (hunk 1 above)
Comment (vishwesh3) [inline src/api/starMessage.js]: Done :+1:

**Comment by vishwesh3** [comment]
Updated branch.
Added another commit: https://github.com/zulip/zulip-mobile/pull/715/commits/e2e4d2c7725ac505e22ebbd3997eda613bf82a3e which resolve app crash when this objects (flags) are accessed (mainly on freshly installed app).

**Comment by borisyankov** [comment]
Looks good. Thanks.

**Pull request merged**